docs/notebooks/delayed-reward.ipynb

```diff
@@ -532,7 +757,7 @@
             "name": "python",
             "nbconvert_exporter": "python",
             "pygments_lexer": "ipython3",
-            "version": "3.9.16"
+            "version": "3.12.6"
```

Comment:
Currently, all development is done on Python=3.10.15. Do you mind re-running this in a 3.10 environment?

Reply:
Done!